src/components/v5/common/ActionSidebar/partials/TeamsSelect/TeamsSelect.tsx

```diff
@@ -38,7 +38,7 @@ const TeamsSelect: FC<TeamSelectProps> = ({
     },
   ] = useToggle();
   const selectedOption = teamsOptions.find(
-    (option) => option.value === selectedTeam,
+    (option) => option.value === Number(selectedTeam),
```

Comment:
Is it possible for saving the value as a number directly?

Reply:
As above. Id now is passed as number to EditTeamForm